deb/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        build-essential \
+        dh-make \
+        devscripts \
+        equivs \
+        lintian
```

Comment:
It's recommend to clean the apt cache afterwards: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/apt-get